src/zjs_spi.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017, Intel Corporation.
+
+#ifndef __zjs_spi_h__
+#define __zjs_spi_h__
+
+#include "jerryscript.h"
+
+enum spi_topology {
+    ZJS_TOPOLOGY_FULL_DUPLEX,
+    ZJS_TOPOLOGY_SINGLE_READ,
+    ZJS_TOPOLOGY_SINGLE_WRITE,
```

Comment:
I have renamed this to TOPOLOGY_Write because it may not be a single write. In some doc it figures like "single write" though.